server/api/users.js

```diff
@@ -47,14 +48,41 @@ router.post("users.list", auth(), pagination(), async (ctx) => {
     limit: ctx.state.pagination.limit,
   });
 
+  const total = await User.count({ where });
+
   ctx.body = {
-    pagination: ctx.state.pagination,
+    pagination: {
+      ...ctx.state.pagination,
+      total,
+    },
     data: users.map((listUser) =>
       presentUser(listUser, { includeDetails: user.isAdmin })
     ),
   };
 });
 
+router.post("users.count", auth(), async (ctx) => {
+  const user = ctx.state.user;
+  const users = await User.findAll({
```

Comment:
Chatted with Nan about this, it was a red flag because you're querying every single user on the team and loading them into memory here.

We think that this is actually a perfect spot to break out of the ORM and just do a raw query to count up the various types of user inside PG. Here's an example:

https://dba.stackexchange.com/a/23074